AcademicReward/Database/AcademicRewardsDatabase.cs

```diff
@@ -13,7 +13,7 @@ public class AcademicRewardsDatabase {
     private const string BitApiKey = "v2_3vaWv_9N4SgRBSZweqRCh5mKAQ4BE";
     private const string BitUsername = "pattmax";
     private const string BitDatabaseName = "pattmax/AcademicRewards";
-    private string connectionString = string.Empty;
+    private string _connectionString = string.Empty;
```

Comment:
Were these underscores intended? Should these variable names really have an underscore at the start? I see quite a few of them regarding our database stuff as well. Some of our property values have it too

Reply:
Yes, local private variables should be prefixed with an underscore

Reply:
https://stackoverflow.com/questions/3136594/naming-convention-underscore-in-c-and-c-sharp-variables

Reply:
I had no idea, looks good to me